PowerShellTools/PowerShellToolsPackage.cs

```diff
@@ -193,6 +204,8 @@ private void RefreshCommands(params ICommand[] commands)
 
         private void InitializeInternal()
         {
+            base.Initialize();

```

Comment:
A bit odd to have InitializeInternal call base.Initialize.  Why not within Initialize()?